src/Controls/samples/Controls.Sample/Pages/Controls/ShapesGalleries/ShapesGallery.cs

```diff
@@ -26,7 +26,7 @@ public ShapesGallery()
 						GalleryBuilder.NavButton("AutoSize Shapes Gallery", () => new AutoSizeShapesGallery(), Navigation),
 						GalleryBuilder.NavButton("Path Gallery", () => new PathGallery(), Navigation),
 						GalleryBuilder.NavButton("Path Aspect Gallery", () => new PathAspectGallery(), Navigation),
-						GalleryBuilder.NavButton("Path LayoutOptions Gallery", () => new PathLayoutOptionsGallery(), Navigation),
+						GalleryBuilder.NavButton("Shapes LayoutOptions Gallery", () => new PathLayoutOptionsGallery(), Navigation),
```

Comment:
Not related with the fix but, just updated this sample to include all the shapes and validate the alignment:

![image](https://user-images.githubusercontent.com/6755973/161080937-814b2adf-1a99-49c3-80b7-c2c0bb177dcd.png)